eng/Versions.props

```diff
@@ -248,11 +248,11 @@
     <GrpcAuthVersion>2.45.0</GrpcAuthVersion>
     <GrpcNetClientVersion>2.45.0</GrpcNetClientVersion>
     <GrpcToolsVersion>2.45.0</GrpcToolsVersion>
-    <DuendeIdentityServerAspNetIdentityVersion>6.0.4</DuendeIdentityServerAspNetIdentityVersion>
-    <DuendeIdentityServerEntityFrameworkVersion>6.0.4</DuendeIdentityServerEntityFrameworkVersion>
-    <DuendeIdentityServerVersion>6.0.4</DuendeIdentityServerVersion>
-    <DuendeIdentityServerStorageVersion>6.0.4</DuendeIdentityServerStorageVersion>
-    <DuendeIdentityServerEntityFrameworkStorageVersion>6.0.4</DuendeIdentityServerEntityFrameworkStorageVersion>
+    <DuendeIdentityServerAspNetIdentityVersion>6.1.0</DuendeIdentityServerAspNetIdentityVersion>
```

Comment:
@premun could we get the 6.1.0 versions of these packages uploaded to the feed?

Reply:
Done